setup.cfg

```diff
@@ -68,7 +68,10 @@ scripts_are_modules = True
 no_implicit_optional = True
 warn_redundant_casts = True
 show_error_codes = True
-enable_error_code = ignore-without-code
+# See python/mypy Issue #10709
```

Comment:
I think we're not ready to activate mypy yet (it's in precommit but everything is deactivated), it's better if we don't mix it with another change.